rts/builds/headless/CMakeLists.txt

```diff
@@ -28,13 +28,16 @@ list(APPEND engineHeadlessLibraries no-sound)
 list(APPEND engineHeadlessLibraries engineSim)
 list(APPEND engineHeadlessLibraries pr-downloader)
 list(APPEND engineHeadlessLibraries RmlCore)
+list(APPEND engineHeadlessLibraries RmlDebugger)
+list(APPEND engineHeadlessLibraries RmlSolLua)
+list(APPEND engineHeadlessLibraries lunasvg)
```

Comment:
you shouldn't need this line either, here and below, it should be correctly transitively linked via RmlCore

Reply:
Without the lunasvg one I get linked errors:
```
: && /usr/lib64/ccache/g++ -fuse-ld=gold -std=c++20 -fdiagnostics-color=auto -march=x86-64 -mtune=generic -msse -mno-sse3 -mno-ssse3 -mno-sse4.1 -mno-sse4.2 -mno-sse4 -mno-sse4a -mno-avx -mno-fma -mno-fma4 -mno-xop -mno-lwp -mno-avx2  -mfpmath=sse -fsingle-precision-constant -frounding-math -mieee-fp -pipe -fno-strict-aliasing  -fvisibility=hidden  -fvisibility-inlines-hidden -pthread -O3 -g -DNDEBUG -fdiagnostics-color=always  -Wformat -Wformat-security -DNDEBUG -fuse-ld=lld  -Wl,--icf=all  -Wl,-O3    -rdynamic rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Action.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/AviVideoCapturing.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/CameraController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/FPSController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/FreeController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/OverheadController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/OverviewController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/RotOverheadController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Camera/SpringController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/CameraHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/ChatMessage.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/ClientData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/ClientSetup.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/CommandMessage.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Console.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/ConsoleHistory.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/DummyVideoCapturing.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/FPSUnitController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Game.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameControllerTextInput.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameHelper.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameSetup.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GameVersion.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/GlobalUnsynced.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/IVideoCapturing.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/InMapDraw.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/InMapDrawModel.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/LoadScreen.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Players/Player.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Players/PlayerBase.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Players/PlayerHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Players/PlayerStatistics.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/Players/TeamController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/PreGame.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/SelectedUnitsHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/SelectedUnitsAI.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/SyncedGameCommands.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/TraceRay.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/CommandColors.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/CursorIcons.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/EndGameBox.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/GameInfo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/GameSetupDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/GuiHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/Groups/Group.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/Groups/GroupHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/HwMouseCursor.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/InfoConsole.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/InputReceiver.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/KeyBindings.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/KeyCodes.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/ScanCodes.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/IKeys.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/KeySet.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/MiniMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/MouseCursor.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/MouseHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/PlayerRoster.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/PlayerRosterDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/ProfileDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/QuitBox.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/ResourceBar.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/SelectionKeyHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/ShareBox.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/StartPosSelecter.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/TooltipConsole.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UI/UnitTracker.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/UnsyncedGameCommands.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/WaitCommandsAI.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Game/WordCompletion.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/AutohostInterface.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/GameServer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/GameParticipant.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/Protocol/BaseNetProtocol.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/Protocol/NetProtocol.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Net/NetCommands.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaArchive.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaBitOps.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstCMD.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstCMDTYPE.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstCOB.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstGL.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstEngine.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstGame.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaConstPlatform.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVFSDownload.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaFBOs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaFeatureDefs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaFonts.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaGaia.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaHandle.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaHandleSynced.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaIO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaInputReceiver.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaInterCall.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaIntro.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaMaterial.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaMathExtra.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaMemPool.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaMenu.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaMetalMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaObjectRendering.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaOpenGL.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaOpenGLUtils.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaPathFinder.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaRBOs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaRules.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaRulesParams.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaScream.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaShaders.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaSyncedCtrl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaSyncedMoveCtrl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaSyncedRead.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaSyncedTable.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaTextures.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaAtlasTextures.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUI.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUICommand.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUnitDefs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUnsyncedCtrl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUnsyncedRead.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaUtils.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVFS.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaWeaponDefs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaZip.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVAO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVAOImpl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVBO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Lua/LuaVBOImpl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/BaseGroundDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/BasicMapDamage.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/Ground.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/HeightLinePalette.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/HeightMapTexture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/MapDamage.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/MapInfo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/MapParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/MetalMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/ReadMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/Generation/BlankMapGenerator.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/SMFGroundDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/SMFGroundTextures.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/SMFMapFile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/SMFReadMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/SMFRenderState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/Basic/BasicMeshDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/ROAM/Patch.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Map/SMF/ROAM/RoamMeshDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/AVIGenerator.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/DebugColVolDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/DebugVisibilityDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/DebugDrawerAI.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/QTPFSPathDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/HAPFSPathDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/AdvWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/IGroundDecalDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/ISky.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/IWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/BasicWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/BumpWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/CubeMapHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/DebugCubeMapTexture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/DynWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/GrassDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/MapRendering.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/ModernSky.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/NullSky.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/RefractWater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/SkyBox.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/SkyLight.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/SunLighting.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/WaterRendering.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Decals/GroundDecalHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Decals/DecalsDrawerGL4.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Decals/LegacyTrackHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/ProjectileDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/BitmapMuzzleFlame.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/BubbleProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/DirtProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/ExploSpikeProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/FlyingPiece.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/GeoSquareProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/GeoThermSmokeProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/NanoProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/HeatCloudProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/MuzzleFlame.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/RepulseGfx.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/SimpleParticleSystem.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/ShieldSegmentProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/SmokeProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/SmokeProjectile2.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/SmokeTrailProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/SpherePartProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/TracerProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/WakeProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Env/Particles/Classes/WreckProjectile.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/FBO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/StreamBuffer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/GeometryBuffer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/FixedPipelineState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/glStateDebug.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/LightHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/VertexArray.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/VertexArrayTypes.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/RenderBuffers.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/VBO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/VAO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/glExtra.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/State.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GL/myGL.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GlobalRendering.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/GroundFlash.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/CommandDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/HUDDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/IPathDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/IconHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/InMapDrawView.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/LineDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/LuaObjectDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/SmoothHeightMeshDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/InfoTexture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/IInfoTextureHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Legacy/LegacyInfoTextureHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/AirLos.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Combiner.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Height.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/InfoTextureHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Los.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Metal.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/MetalExtraction.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Path.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/PboInfoTexture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Map/InfoTexture/Modern/Radar.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Materials/CustomMaterial.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Materials/CustomMaterialRenderer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/3DModel.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/3DModelVAO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/3DOParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/AssIO.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/AssParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/IModelParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/S3OParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/ModelsMemStorageDefs.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Models/ModelsMemStorage.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Screenshot.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Shaders/GLSLCopyState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Shaders/LuaShaderContainer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Shaders/Shader.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Shaders/ShaderHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Shaders/ShaderStates.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/ShadowHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/TeamHighlight.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/3DOTextureHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/Bitmap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/ColorMap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/LegacyAtlasAlloc.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/NamedTextures.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/S3OTextureHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/TAPalette.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/TextureFormat.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/TextureAtlas.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/TexturesSet.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/nv_dds.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/QuadtreeAtlasAlloc.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Textures/RowAtlasAlloc.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Common/ModelDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Common/ModelDrawerData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Common/ModelDrawerState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Common/ModelDrawerHelpers.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Features/FeatureDrawerData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Features/FeatureDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Units/UnitDrawerData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Units/UnitDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/ModelsDataUploader.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/OGLDBInfo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/UniformConstants.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/VerticalSync.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/WorldDrawer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Fonts/CFontTexture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Fonts/glFont.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Fonts/glFontRenderer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rendering/Fonts/TextWrap.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/Button.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/Gui.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/GuiElement.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/HorizontalLayout.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/LineEdit.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/List.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/Picture.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/TextElement.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/VerticalLayout.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/aGui/Window.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rml/Backends/RmlUi_Backend.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rml/Backends/RmlUi_Platform_SDL.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Rml/Backends/RmlUi_Renderer_GL3.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Menu/LuaMenuController.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Menu/SelectMenu.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/Menu/SelectionWidget.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/AABB.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/AIScriptHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Color.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Config/ConfigHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Config/ConfigLocater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Config/ConfigSource.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Config/ConfigVariable.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/CRC.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/EventClient.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/EventHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/GlobalConfig.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Info.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Input/InputHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Input/KeyInput.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Input/MouseInput.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/CregLoadSaveHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/Demo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/DemoReader.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/DemoRecorder.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/LoadSaveHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LoadSave/LuaLoadSaveHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/LogOutput.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Main.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Matrix44f.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Misc/RectangleOverlapHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Misc/SpringTime.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Object.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/GameLoadThread.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Option.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Clipboard.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/errorhandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Misc.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/SharedLib.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/ScopedFileLock.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/SDL1_keysym.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Watchdog.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/WindowManagerHelper.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Rectangle.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SafeVector.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SafeCStrings.c.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SplashScreen.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SpringApp.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/StartScriptGen.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/DumpState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/FPUCheck.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/Logger.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/SHA512.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/SyncChecker.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/SyncDebugger.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/SyncedFloat3.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/backtrace.c.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Sync/get_executable_name.c.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/TdfParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Threading/ThreadPool.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/TimeProfiler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/TimeUtil.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/UriParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/StringHash.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/StringUtil.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/type2.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/float3.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/float4.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SpringMath.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/SpringMem.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/creg/SerializeLuaState.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/creg/Serializer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/creg/VarTypes.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/creg/creg.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/creg/creg_runtime_tests.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/ArchiveNameResolver.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/ArchiveLoader.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/ArchiveScanner.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/CacheDir.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/DataDirLocater.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/DataDirsAccess.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/FileFilter.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/FileHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/FileSystem.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/FileSystemAbstraction.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/FileSystemInitializer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/GZFileHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/RapidHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/SimpleParser.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/FileSystem/VFSHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/Backend.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/DefaultFilter.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/DefaultFormatter.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/FramePrefixer.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/LogSinkHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/LogUtil.c.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/TracySink.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/ConsoleSink.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Log/FileSink.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/CrashHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/SoLib.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/MessageBox.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/WindowManagerHelper.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/CpuID.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Threading.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/Futex.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/System/Platform/Linux/ThreadSupport.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AICallback.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AICheats.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AIInterfaceKey.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AIInterfaceLibrary.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AIInterfaceLibraryInfo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AILibraryManager.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/AISCommands.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/EngineOutHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/LuaAIImplHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SAIInterfaceCallbackImpl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SSkirmishAICallbackImpl.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAIData.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAIHandler.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAIKey.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAILibrary.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAILibraryInfo.cpp.o rts/builds/legacy/CMakeFiles/engine-legacy.dir/__/__/ExternalAI/SkirmishAIWrapper.cpp.o -o spring  -Wl,-Bstatic  -lSDL2main  -lSDL2main  -Wl,-Bdynamic  -lSDL2  -lGL  -lGLU  -lGLEW  -lfontconfig  -lfreetype  -lXcursor  -lX11  rts/lib/squish/libsquish.a  rts/lib/rg-etc1/librgetc1.a  rts/System/Sound/libsound.a  rts/System/Net/libengineSystemNet.a  rts/lib/libcpuid/libcpuid/libcpuid.a  rts/lib/smmalloc/libsmmalloc.a  -ldl  -lIL  tools/pr-downloader/src/lib/7z/lib7zip.a  -lminizip  -lz  rts/lib/tracy/libTracyClient.a  rts/lib/lua/liblua.a  rts/lib/luasocket/libluasocket.a  rts/System/FileSystem/Archives/libarchives.a  rts/lib/assimp/code/libassimp.a  rts/lib/gflags/libgflags_nothreads.a  rts/lib/streflop/libstreflop.a  -lunwind  -llzma  -Wl,-whole-archive  rts/Sim/libengineSim.a  -Wl,-no-whole-archive  rts/Sim/libengineSim.a  tools/pr-downloader/src/libpr-downloader.a  rts/lib/RmlUi/libRmlCore.a  rts/lib/RmlUi/libRmlDebugger.a  rts/lib/RmlSolLua/libRmlSolLua.a  -lvorbisfile  -lvorbis  -logg  -lopenal  rts/lib/assimp/contrib/irrXML/libIrrXML.a  tools/pr-downloader/src/lib/7z/lib7zip.a  -ljsoncpp  -lcurl  /usr/lib64/libz.so  -lminizip  tools/pr-downloader/src/lib/md5/libpr-md5.a  tools/pr-downloader/src/lib/sha1/libpr-sha1.a  rts/lib/RmlUi/libRmlCore.a  -lfreetype  rts/lib/tracy/libTracyClient.a  -ldl && :
ld.lld: error: undefined symbol: lunasvg::Document::~Document()
>>> referenced by unique_ptr.h:99 (/usr/include/c++/13/bits/unique_ptr.h:99)
>>>               ElementSVG.cpp.o:(Rml::ElementSVG::~ElementSVG()) in archive rts/lib/RmlUi/libRmlCore.a
>>> referenced by unique_ptr.h:99 (/usr/include/c++/13/bits/unique_ptr.h:99)
>>>               ElementSVG.cpp.o:(Rml::ElementSVG::~ElementSVG()) in archive rts/lib/RmlUi/libRmlCore.a
>>> referenced by unique_ptr.h:99 (/usr/include/c++/13/bits/unique_ptr.h:99)
>>>               ElementSVG.cpp.o:(Rml::ElementSVG::LoadSource()) in archive rts/lib/RmlUi/libRmlCore.a
>>> referenced 2 more times
```
Maybe I messed something up somewhere else?

Reply:
Also similar kinds of errors without RmlDebugger

Reply:
Ok, let's keep it then